c2corg_ui/locale/fr/LC_MESSAGES/c2corg_ui-client.po

```diff
@@ -1282,7 +1302,7 @@ msgstr "Remontées mécaniques"
 #: c2corg_ui/templates/outing/edit.html
 #: c2corg_ui/templates/outing/helpers/detailed_outings_attributes.html
 msgid "lift_status"
-msgstr ""
+msgstr "Etat d'ascenseur"
```

Comment:
well, gotta know that "lift (ascenseur) status (etat)" is translated to rémontées mécaniques :D